modules/core/core-api/src/main/java/com/enonic/xp/query/aggregation/TermsAggregationQuery.java

```diff
@@ -50,7 +50,7 @@ public Type getOrderType()
         return orderType;
     }
 
-    public long getMinDocCount()
+    public Long getMinDocCount()
```

Comment:
This is impossible to change return type in API.

Reply:
Done